### DISCUSSION TITLE
How do I level up my CSS skills here?

### DISCUSSION THREAD

**Discussion by ImagenMan** [cssbattle/forum#389] [Q&A]
I enjoy this app and have made a routine of completing the daily challenges. I feel fairly confident in completing them in a timely manner and with what I consider atm using few characters. I feel I have improved in my overall skills. Having looked at other answers from other talented members, I am having a hard time grasping their techniques or angles. I seem to mostly use my same techniques over and over. It works but... I see others codes and they are accomplishes their codes in 50% less characters than I can. I know that you may be reluctant to "give away" your tricks that help you be at the top but any guidance or tips or suggestions is greatly appreciated.  In the end I am searching for a way to grow in CSS. Thanks for your time

**Comment by chinchang** [comment]
Our Discord is one place where lot of discussions happen over the targets. Lot of top players often share insights and explanations about the tricks they use too. https://discord.gg/rvqvbcFM86

**Reply by ImagenMan** [reply]
I finally got my Discord linked. Gonna go check it out
